cobalt/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java

```diff
@@ -104,6 +104,22 @@ protected void createContent(final Bundle savedInstanceState) {
       // Initializing the command line must occur before loading the library.
       if (!CommandLine.isInitialized()) {
           ((CobaltApplication) getApplication()).initCommandLine();
+
+          // Note that appendSwitchesAndArguments excludes cobaltCommandLineParams[0]
+          // as the program name, and all other arguments SHOULD start with '--'.
+          String[] cobaltCommandLineParams = new String[]{"",
+          // disable first run experience
```

Comment:
: ). my only (pedantic) nit woudl be to write these as full sentences:
```
  // Disable ... experience.
```

(Uppercase, then period at the end).  
https://google.github.io/styleguide/cppguide.html#Punctuation,_Spelling_and_Grammar